### PR TITLE
Added resources under some categories(changes made)

### PR DESCRIPTION
**Pull request by MugdhaBehere**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description

Added resources under the following categories:
Interactive Tutorials: SQLZoo
Free English Courses: Stanford CS224N, Stanford CS230
Free Programming books: The official Swift book by Apple titled "The Swift Programming Language"
Free Programming Cheat Sheet: The Big-O Cheat Sheet.

### Why is this valuable (or not)?

SQLZoo is interactive and free.
CS224N and CS230 are courses by Stanford University which makes it valuable.
The Swift Programming Language is an official resource and free.
The Big-O Cheat Sheet is comprehensive and covers the complexities of many algorithms.

### How do we know it's really free?

We can browse the complete website of SQLZoo for free without encountering any webpage that asks us to pay.
Stanford CS224N and Stanford CS230 are courses with their content released and their content available for free on Stanford's website and YouTube Channel.
The Swift Programming Language which is the official Swift book by Apple is made available under a [Creative Commons Attribution 4.0 International (CC BY 4.0) License] which means that it is free.
The website for Big-O Cheat Sheet can also be completely browsed for free without encountering any webpage that asks us to pay.

### For book lists, is it a book? For course lists, is it a course? etc.

SQLZoo is an interactive tutorial.
Stanford CS224N and CS230 are courses
The Swift Programming Language is an ebook.
The Big-O Cheat Sheet is a Cheat Sheet.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -529,6 +530,7 @@
 * [Introduction to Reinforcement learning with David Silver](https://www.youtube.com/playlist?list=PLqYmG7hTraZBiG_XpjnPrSNw-1XQaM_gB) - David Silver
 * [MIT 6.S094: Deep Learning for Self-Driving Cars](https://selfdrivingcars.mit.edu)
 * [MIT 6.S191: Introduction to Deep Learning](http://introtodeeplearning.com) - Alexander Amini, Ava Soleimany
+* [Natural Language Processing with Deep Learning - Stanford CS 224N](https://www.youtube.com/playlist?list=PLoROMvodv4rOSH4v6133s9LFPRHjEmbmJ) (Stanford University)
```
Comment (eshellman) [inline courses/free-courses-en.md:533]:
```suggestion
* [Stanford CS 224N: Natural Language Processing with Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOSH4v6133s9LFPRHjEmbmJ) (Stanford University)
```
Don't invent titles! Use the title given by the resource.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -516,6 +516,7 @@
 * [Advanced NLP using spaCy](https://course.spacy.io) - Ines Montani
 * [Complete Deep Learning](https://www.youtube.com/playlist?list=PLZoTAELRMXVPGU70ZGsckrMdr0FteeRUi) - Krish Naik
 * [Convolutional Neural Networks for Visual Recognition](http://cs231n.github.io)
+* [Deep Learning Autumn 2018 - Stanford CS 230](https://www.youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb) (Stanford University)
```
Comment (eshellman) [inline courses/free-courses-en.md:519]:
```suggestion
* [Stanford CS230: Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb) (Stanford University)
```

**Review by eshellman — CHANGES_REQUESTED** [review]
changing whitespace make review more difficult and open the PR to conflicts during the review. Happy to remove the extra whitespace, but easier to review if it's a separate PR.
when you've made changes, please add `(changes made)` to the title of your PR

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -2500,6 +2500,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [The Swift Programming Language](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html) (HTML)
     * [The Swift Programming Language (Swift 5.7)](https://itunes.apple.com/us/book/swift-programming-language/id881256329?mt=11) - Apple Inc. (iBook)
     * [Using Swift with Cocoa and Objective-C](https://itunes.apple.com/us/book/using-swift-cocoa-objective/id888894773?mt=11) - Apple Inc. (iBook)
+* [The Swift Programming Language (Official Swift book by Apple)](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/) (HTML)
```
[the remark this answers is not in the document]
Comment (MugdhaBehere) [reply books/free-programming-books-langs.md]: I have removed this, created a new branch, and a new Pull Request which has been numbered #9846.

File: courses/free-courses-en.md (hunk 1 above)
Comment (MugdhaBehere) [reply courses/free-courses-en.md]: This change was also made, creating a new branch, and a new Pull Request which has been numbered #9846.

File: courses/free-courses-en.md (hunk 2 above)
Comment (MugdhaBehere) [reply courses/free-courses-en.md]: This change was also made, creating a new branch, and a new Pull Request which has been numbered #9846. Please review.

**Pull request closed without merging**